uobtheatre/bookings/test/factories.py

```diff
@@ -61,7 +61,7 @@ class Meta:
 
 class BookingFactory(factory.django.DjangoModelFactory):
 
-    reference = create_short_uuid()
+    reference = factory.LazyAttribute(lambda _: create_short_uuid())
```

Comment:
What happens if we just remove this line?

Reply:
Guessing that's been left in error!

Reply:
the world keeps on spinning

Reply:
:partying_face: